examples/cluster/plot_digits_linkage.py

```diff
@@ -35,15 +36,18 @@
 
 np.random.seed(0)
 
+
+def shift(x):
+    return ndimage.shift(x.reshape((8, 8)),
+                  .3 * np.random.normal(size=2),
+                  mode='constant').ravel()
```

Comment:
This indentation level looks weird to me. Also does it fit on a single line? It should fit on two at most:

```python
def shift(x):
    return ndimage.shift(x.reshape((8, 8)), .3 * np.random.normal(size=2),
                         mode='constant').ravel()

```